clusters/build-clusters/01_cluster/cert-manager/registry-build01_certificate.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  labels:
+    gcp-project: openshift-ci-infra
+  name: registry-build01
+  namespace: openshift-image-registry
+spec:
+  secretName: public-route-tls
+  issuerRef:
+    name: cert-issuer
+    kind: ClusterIssuer
+  dnsNames:
+  - registry.build01.ci.openshift.org
```

Comment:
Why do we create a certificate rather than adding an annotated Ingress?

Reply:
There is no ingress in this case :)

Reply:
the route is created by the registry when we change its config.